### PR TITLE
Add metadata command option to skip markup triple slash comments

### PR DESCRIPTION
**Pull request by qinezh**
#756 @superyyrrzz @hellosnow @ansyral @vwxyzh @chenkennt 

Documentation will be updated in the next PR

### CODE REVIEW

File: src/docfx/Models/MetadataJsonItemConfig.cs [hunk 1]
```diff
@@ -16,6 +16,9 @@ public class MetadataJsonItemConfig
         [JsonProperty("force")]
         public bool? Force { get; set; }
 
+        [JsonProperty("skip")]
```
Comment (hellosnow) [inline src/docfx/Models/MetadataJsonItemConfig.cs]: skipMarkup?

File: src/Microsoft.DocAsCode.Build.ManagedReference/BuildManagedReferenceDocument.cs [hunk 2]
```diff
@@ -47,12 +47,19 @@ public override void Build(FileModel model, IHostService host)
         public static ItemViewModel BuildItem(IHostService host, ItemViewModel item, FileModel model, Func<string, bool> filter = null)
         {
             var linkToUids = new HashSet<string>();
-            item.Summary = Markup(host, item.Summary, model, filter);
-            item.Remarks = Markup(host, item.Remarks, model, filter);
-            if (model.Type != DocumentType.Overwrite)
+            var pageViewModel = model.Content as PageViewModel;
+            var skip = pageViewModel?.IsSkipMarkup;
+
+            if (skip == null || skip.Value != true)
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.ManagedReference/BuildManagedReferenceDocument.cs]: >skip == null || skip.Value != true [](start = 16, length = 34)

skip != true

File: src/docfx/Models/MetadataCommandOptions.cs [hunk 3]
```diff
@@ -16,6 +16,9 @@ internal class MetadataCommandOptions : ICanPrintHelpMessage, ILoggable
         [Option('f', "force", HelpText = "Force re-generate all the metadata")]
         public bool ForceRebuild { get; set; }
 
+        [Option("skipMarkup", HelpText = "Skip to markup the triple slash comments")]
+        public bool IsSkipMarkup { get; set; }
```
Comment (ansyral) [inline src/docfx/Models/MetadataCommandOptions.cs]: >IsSkipMarkup [](start = 20, length = 12)

SkipMarkup?

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/ExtractMetadataInputModel.cs [hunk 4]
```diff
@@ -24,6 +24,8 @@ public class ExtractMetadataInputModel
 
         public bool ForceRebuild { get; set; }
 
+        public bool IsSkipMarkup { get; set; }
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/ExtractMetadataInputModel.cs]: also update to "should"?

**Pull request merged**